modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,396 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer Operator
+You can install an AWS Load Balancer Operator (ALBO) if you meet certain requirements.
+
+.Prerequisites
+
+* You have an existing
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA)
+endif::openshift-rosa[]
+cluster with bring-your-own-VPC (BYO-VPC) configuration across multiple availability zones (AZ) installed in STS mode.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to modify the VPC and subnets of the created
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster.
+// * AWS account credentials
+* You have installed the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+CLI (`rosa`).
+* You have installed the Amazon Web Services (AWS) CLI.
+* You have installed the OpenShift CLI (oc).
+* You are using OpenShift Container Platform (OCP) 4.13 or later.
+
+[IMPORTANT]
+====
+When installing an ALBO for use with
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+a ROSA
+endif::openshift-rosa[]
+cluster in an AWS Local Zone (LZ), you must enable the AWS LZ for the account, and AWS Elastic Load Balancing v2 (ELBv2) services must be available in the AWS LZ.
+====
+.Procedure
+
+. Identify the cluster infrastructure ID and the cluster OpenID Connect (OIDC) DNS by running the following commands:
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster INFRA ID:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'Infra ID'
+----
++
+or
++
+[source,terminal]
+----
+$ oc get infrastructure cluster -o json | jq -r '.status.infrastructureName'
+----
+
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster OIDC DNS:
++
+--
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'OIDC'
+----
+
+Save the output from both commands. You will use this information in future steps within this procedure.
+
+--
+. Create the AWS IAM policy required for the ALBO:
++
+.. Create ALBO namespace:
++
+[source,terminal]
+----
+$ oc create namespace aws-load-balancer-operator
+----
++
+.. Assign the following trust policy to the newly-created AWS IAM role. For more information on assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
++
+[source,json]
```

Comment:
I'm not sure, but is this block in json format or something else? It's rendering in a weird way in the preview.

Reply:
Should have been [source,terminal] in both instances.